modules/api/src/main/java/org/eclipse/xpanse/api/AdminServicesApi.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.api;
+
+import static org.eclipse.xpanse.modules.models.security.constant.RoleConstants.ROLE_ADMIN;
+import static org.eclipse.xpanse.modules.models.security.constant.RoleConstants.ROLE_CSP;
+import static org.eclipse.xpanse.modules.models.security.constant.RoleConstants.ROLE_USER;
+
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.annotation.Resource;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.xpanse.modules.models.admin.IdentityProviderStatus;
+import org.eclipse.xpanse.modules.models.admin.SystemStatus;
+import org.eclipse.xpanse.modules.models.admin.enums.HealthStatus;
+import org.eclipse.xpanse.modules.models.admin.enums.IdentityProviderType;
+import org.eclipse.xpanse.modules.security.zitadel.ZitadelAuthorizationService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationContext;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Admin services Api.
+ */
+@Slf4j
+@CrossOrigin
+@RestController
+@RequestMapping("/xpanse")
+@Secured({ROLE_ADMIN})
+public class AdminServicesApi {
+
+
+    @Resource
+    private ApplicationContext applicationContext;
+
+    @Value("${spring.profiles.active:default}")
+    private String activeProfiles;
+
+    /**
+     * Method to find out the current state of the system.
+     *
+     * @return Returns the current state of the system.
+     */
+    @Tag(name = "Admin", description = "APIs for administrating Xpanse")
+    @GetMapping(value = "/health", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.OK)
+    @Secured({ROLE_ADMIN, ROLE_CSP, ROLE_USER})
```

Comment:
@baixinsui 
I think I made a mistake while analyzing this topic. 
For  the 'user' and 'csp' role, we must only return the 'SystemStatus' without the status of other integrated systems such as  Identitiy provider. 

Only for role 'admin' we must return full healthcheck response with IdentityProviderStatus. 

If you agree, we can rename the old `health` method to `systemStatus` which returns only SystemStatus without IdentityProviderStatus. 
add new method healthCheck with a new data model 'HealthCheck' which has SystemStatus + IdentityProviderStatus

Reply:
Ok. I will restore the old method with path 'system/status'  for 'csp' and 'role', update the new method for 'admin'

Reply:
@baixinsui in my opinion, the end users and csp users need not know internal details of the implementation. 
we can show them only IAM status is OK or NOK. But end point details need not be revelated to them.

Reply:
Ok. I don't think it's necessary to add new method. In this method, we can obtain the roles of current user. If roles contains' admin', we will return endpoint and detail information